tutorials/tour/_posts/2017-02-13-operators.md

```diff
@@ -10,8 +10,40 @@ num: 30
 next-page: automatic-closures
 previous-page: local-type-inference
 ---
+In Scala, operators are methods. For example, `+` can be called with dot-notation:
+```
+10.+(1)
+```
+
+However, it more closely resembles math if you write it as an infix operator:
+```
+10 + 1
+```
+
+You can also give `+` specific meaning in your class:
+```tut
+class Vector(val x: Double, val y: Double) {
```

Comment:
Why don't you call this `Vec` to avoid collision with `scala.collection.immutable.Vector` (whose name is really unfortunate for anyone thinking about mathematical vectors)?